api/api_without_chat_id.yaml

```diff
@@ -0,0 +1,214 @@
+openapi: "3.0.2"
+info:
+  title: Chat Dancer
+  version: "1.0"
+servers:
+  - url: 'https://dancier.net'
+  - url: 'http://localhost:8080'
+paths:
+  /user/{user_id}/chats:
+    get:
+      parameters:
+        - $ref: '#/components/parameters/user_id'
+      summary: returns a list of all chats of a user
+      responses:
+        '200':
+          description: "list of chats for a user"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/chat_list_response'
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/error'
+  /user/{user_id}/chats/{partner_id}/messages:
+    get:
```

Comment:
Already written in PM: I would guess that this will have a breaking API when we switch to allow also group chats.
So as you thought yesterday of a solution with something like a room: it would vote for it:
https://stackoverflow.com/questions/46484989/database-schema-for-chat-private-and-group

Reply:
yes, thanks for the schema, it goes in the direction I also thought about